intranet/apps/users/models.py

```diff
@@ -300,7 +307,7 @@ def tj_email(self):
 
         for email in self.emails.all():
             if email.address.endswith(("@fcps.edu", "@tjhsst.edu")):
-                return email
+                return email.address
```

Comment:
Maybe return `str(email)`?

Reply:
I specifically wanted to do `email.address` to be as explicit as possible. In general, I don't like relying on methods like `__str__()`, where the developer can basically pick anything they want it to return and it's hard to argue in a particular direction.